src/Engine/GameScenario.java

```diff
@@ -37,12 +37,18 @@ public GameEventQueue initTurn(GameMap map)
 
   public enum TagMode
   {
-    OFF(false), AWBW(true), Team_Merge(false); // Cartridge, Persistent,
+    OFF       (false, "Dunno what you expected"),
+    AWBW      (true , "Multi-CO; only one active at a time"),
+    Persistent(true , "Multi-CO; only one builds or COPs at a time"),
+    Team_Merge(false, "All teams become a single player");
```

Comment:
All teams become a single player? You mean all COs in the same army, right? Does that mean they all get an equal share of power, and all powers are available for use, and all passives are simultaneously active? That could be a bit cray cray.

Also an important element of Persistent (at least if my reading of the code is correct) is that the units keep the attributes of the CO that built them  when control shifts; the CO in command just "owns" whatever is built during his watch (but does this mean other units don't benefit from the active CO's passive?).

Reply:
> All teams become a single player? You mean all COs in the same army, right?

No. You'll note that you can't pick multiple COs in Team Merge.
If I'm on Final Front with Grit, Andy, and Kanbei, and I pick Team Merge, I have *one* turn for all 3 COs.
It also does share money and the "powers benefit everyone" thing, but Kanbei has yellow buildings and builds Kanbei dudes and there's no swapping.
> Does that mean they all get an equal share of power

No. Same charging rules as solo.
> and all powers are available for use

Yes. Simultaneously, if you want.
> and all passives are simultaneously active?

On their individual dudes, yes.
As a side note, "stack D2Ds" is how Commander Wars does tags, and it almost makes less sense than cart tags IMO.
> That could be a bit cray cray.

That's usually the plan.
> Also an important element of Persistent (at least if my reading of the code is correct) is that the units keep the attributes of the CO that built them when control shifts

Yes. That is the primary reason Persistent exists.
I suppose I could have clarified better, probably by specifying more on the AWBW entry.
> the CO in command just "owns" whatever is built during his watch (but does this mean other units don't benefit from the active CO's passive?).

Yes. Colin can save money so Kanbei can buy units, but he can't buy units and make them Kanbei's.

Reply:
These could still be clearer I think, but given that this isn't user facing at this point, I won't sweat it too much.